### PR TITLE
Mayhem Heroes mayhem integration

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Build Stage:
```
Comment (xansec) [inline Dockerfile:1]: Please include a package stage to reduce image size.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Project name should be generic, and you may omit the image name as it is interpolated during the action run. Same with duration.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I've added the requested changes. Let me know if everything looks good. Thanks

**Comment by xansec** [comment]
Approved!

**Pull request merged**